setup-machine-arktos.sh

```diff
@@ -185,18 +134,6 @@ pip3 install PyYAML
 
 ####################
 
-echo Setup: Change Containerd
```

Comment:
It seems that customized containerd with multi-tenant ability is still needed for arktos integration, for the current mizarcni implementation. You can have some explanation why the upstream containerd is used here as temporary placement.

Reply:
I filed issue https://github.com/CentaurusInfra/mizar/issues/433 for tracking.